docs/src/main/sphinx/release/release-384.md

```diff
@@ -0,0 +1,39 @@
+# Release 384 (3 June 2022)
+
+## General
+
+* Fix queries failing due to worker data exchange optimization introduced in
+  Trino 383. ({issue}`12670`)
+* Fix potential failure or incorrect result of `json_query` function. ({issue}`12681`)
+
+## Druid connector
+
+* Add `query` table function for full query pass-through to the connector. ({issue}`1839`,`9163`)
```

Comment:
does that work in terms of output

{issue}`1839`,`9163`

in the past I always did

{issue}`1839`, {issue}`9163`